OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
@fadyanwar There is some documentation about how this works in the wiki [here](https://github.com/OpenRA/OpenRA/wiki/Defining-custom-AI#support-power-targeting).

The documentation has an example about using atomic bombs.

If the example documentation isn't clear enough, please let us know.
I'd love to make it easy to understand for anyone, new to or experienced with OpenRA.